src/components/Modal/index.stories.js

```diff
@@ -44,8 +45,8 @@ class ModalExample extends Component {
         />
 
         <DocSection title='Demo'>
-          <div className='row'>
-            <div className='col-md-3'>
+          <div className='row' style={{ height: '2000px' }}>
```

Comment:
I'm guessing this is so the page itself forces a scrollbar?  If so, can you move this to a standalone div after the `.row`?

Reply:
(Don't want people confused as to why it needs an inline style on the row if they inspect it)

Reply:
good point